src/test/index.js

```diff
@@ -0,0 +1,37 @@
+import chai from 'chai';
```

Comment:
rename this file from `index.js` to `index.spec.js` to differentiate test file from normal js file

Reply:
done